### PR TITLE
`com.intuit.playerui` publishing scope

### PR DESCRIPTION
**Pull request by sugarmanz**
### Change Type (required)
Indicate the type of change your pull request is:

- [ ] `patch`
- [x] `minor`
- [ ] `major`

# Release Notes

Embracing the `player-ui` namespace, the base group ID, and correlating package scopes, have changed:
`com.intuit.player` -> `com.intuit.playerui`

1. Dropping `.jvm` from non-android artifacts
   - This was originally done to leave room for intermediate JS resource artifacts. This is no longer necessary due to improvements in our build process, and enables us to remove this redundancy.

| Artifact | Internal | External |
| -------- | -------- | -------- |
| Headless Player | `com.intuit.player.jvm:core` | `com.intuit.playerui:core` |
| Android Player | `com.intuit.player.android:player` | `com.intuit.playerui:android` |
| Plugins | `com.intuit.player.plugins`<br/>`com.intuit.player.jvm.plugins`<br/>`com.intuit.player.android.plugins` | `com.intuit.playerui.plugins` |

<details>
  <summary>📦 Published PR as canary version: <code>0.6.1--canary.253.9420</code></summary>
  <br />

  Try this version out locally by upgrading relevant packages to 0.6.1--canary.253.9420
</details>

### CODE REVIEW

File: android/player/src/test/java/com/intuit/playerui/android/utils/BrokenAsset.kt [hunk 1]
```diff
@@ -1,17 +1,16 @@
-package com.intuit.player.android.utils
+package com.intuit.playerui.android.utils
 
 import android.view.View
 import android.widget.FrameLayout
 import android.widget.LinearLayout
-import com.eclipsesource.v8.debug.mirror.Frame
```
Comment (brocollie08) [inline android/player/src/test/java/com/intuit/playerui/android/utils/BrokenAsset.kt:6]: this was never used?

**Review by brocollie08 — APPROVED** [review]

File: android/player/src/test/java/com/intuit/playerui/android/utils/BrokenAsset.kt (hunk 1 above)
Comment (sugarmanz) [reply android/player/src/test/java/com/intuit/playerui/android/utils/BrokenAsset.kt:6]: I don't even know what that import is :P I do see an enum `Layout.Frame` below. My guess is it got auto-imported when trying to use that. A little concerning this doesn't break the linting rules.

**Pull request merged**